### PR TITLE
Show ruby version in the prompt for agnoster themes.

### PR DESCRIPTION
**Pull request by janlindblom**
As the title says. Adds the current ruby version (from rvm or rbenv) to the prompt in the agnoster themes.

### CODE REVIEW

File: themes/agnoster/fish_prompt.fish [hunk 1]
```diff
@@ -124,6 +124,23 @@ function prompt_git -d "Display the actual git state"
   end
 end
 
+function prompt_rubies -d 'Display current Ruby (rvm/rbenv)'
+  [ "$theme_display_ruby" = 'no' ]; and return
```
Comment (bucaran) [inline themes/agnoster/fish_prompt.fish]: Of course you want to be concise and differentiate between `"` and `'`, but I hope you find the following more readable:
```fish
if test "$theme_display_ruby" = "no" ]
  return
end
```

File: themes/agnoster/fish_prompt.fish [hunk 2]
```diff
@@ -124,6 +124,23 @@ function prompt_git -d "Display the actual git state"
...
+  set -l ruby_version
+  if type rvm-prompt >/dev/null
+    set ruby_version (rvm-prompt i v g)
+  else if type rbenv >/dev/null
+    set ruby_version (rbenv version-name)
+    # Don't show global ruby version...
+    [ "$ruby_version" = (rbenv global) ]; and return
```
Comment (bucaran) [inline themes/agnoster/fish_prompt.fish]: Same

**Comment by bucaran** [comment]
Thanks! Please check the comments.

**Comment by bucaran** [comment]
@janlindblom Do you mind making the suggested edit, pull rebase and push again? Thanks

**Comment by janlindblom** [comment]
@bucaran yes, sorry for the late reply, I got sick and haven't quite recovered yet. I'll make the edit and resubmit!

**Comment by bpinto** [comment]
ping @janlindblom 
We have moved the theme to their specific repo: https://github.com/oh-my-fish?utf8=%E2%9C%93&query=agnoster

**Pull request closed without merging**